pkg/directory/directory.go

```diff
@@ -0,0 +1,46 @@
+package directory
+
+import "os"
+
+const (
+	cacheTTL = 60.0 // 60 minute TTL
+	workers  = 10
```

Comment:
You should define constants near where they are used. This could also be named more descriptively, ie, "githubFetchWorkers".

Reply:
I was figuring that this would be helpful for other backends as a constant number of maximum works regardless of what it's trying to access. It's more of a generic constant for anything that wants to define the interface.

Reply:
That seems like an implementation detail to me. It took me a minute or two to find where "workers" was defined, which is why I suggested moving it closer to where it's used.

Reply:
Moved.